deepmd/infer/deep_pot.py

```diff
@@ -239,6 +245,11 @@ def eval(
             force_mag = results["energy_derv_r_mag"].reshape(nframes, natoms, 3)
             mask_mag = results["mask_mag"].reshape(nframes, natoms, 1)
             result = (*list(result), force_mag, mask_mag)
+        if "energy_derv_r_derv_r" in list(results.keys()):
+            hessian = results["energy_derv_r_derv_r"].reshape(
+                nframes, 3 * natoms, 3 * natoms
+            )
+            result += (hessian,)
```

Comment:
I prefer the result to have a fixed number of return values regarding what the model is. The Hession can be obtained when a specific argument is set to True, or in another method.

Reply:
It has been modified in the same form as `self.deep_eval.get_has_spin` above.